mhs/outbound/main.py

```diff
@@ -92,15 +94,15 @@ def main():
     ca_certs = "client.pem"
 
     certs_dir.mkdir(parents=True, exist_ok=True)
-    (certs_dir / client_cert).write_text(config.get_config('CLIENT_CERT'))
-    (certs_dir / client_key).write_text(config.get_config('CLIENT_KEY'))
-    (certs_dir / ca_certs).write_text(config.get_config('CA_CERTS'))
+    (certs_dir / client_cert).write_text(secrets.get_secret_config('CLIENT_CERT'))
```

Comment:
We do this process of extracting the certificate environment variables and writing them to file in a few places now. Is it worth extracting this to a utility?